docs/tutorials/platform_beginner/3_customize_the_front_page.md

```diff
@@ -76,11 +76,13 @@ To add a template like this to your site, create a `main_layout.html.twig` file
 
     <title>eZ Platform Beginner Tutorial</title>
 
-    {{  encore_entry_link_tags('tutorial') }}
+    {{ encore_entry_link_tags('tutorial') }}
 
     <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
     <link href='http://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
     <link href='http://fonts.googleapis.com/css?family=Fjalla+One' rel='stylesheet' type='text/css'>
+
+    <script src="{{ asset('bundles/ezplatformadminuiassets/vendors/jquery/dist/jquery.min.js') }}"></script>
```

Comment:
It's safer to include jquery before files that need it. So I would move it before `{{ encore_entry_link_tags('tutorial') }}`